app/.storybook/.env

```
@@ -0,0 +1 @@
+VITE_BASE_PATH = /demo-data-viz
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)